engine/engine/content/builtins/materials/model_skinned_instanced.vp

```diff
@@ -0,0 +1,45 @@
+#version 140
+
+// Positions can be world or local space, since world and normal
+// matrices are identity for world vertex space materials.
+// If world vertex space is selected, you can remove the
+// normal matrix multiplication for optimal performance.
+
+in highp vec4 position;
+in mediump vec2 texcoord0;
+in mediump vec3 normal;
+in mediump vec4 bone_weights;
+in mediump vec4 bone_indices;
+
+// When 'mtx_world' and 'mtx_normal' is specified as attributes,
+// instanced rendering is automatically triggered.
+// To read more about instancing in Defold, navigate to
+// https://defold.com/manuals/material/#instancing
+in mediump mat4 mtx_world;
+in mediump mat4 mtx_normal;
+in mediump vec4 animation_data;
+
+out highp vec4 var_position;
+out mediump vec3 var_normal;
+out mediump vec2 var_texcoord0;
+out mediump vec4 var_light;
+
+uniform vs_uniforms
+{
+    uniform mediump mat4 mtx_view;
+    uniform mediump mat4 mtx_proj;
+    uniform mediump vec4 light;
+};
+
+#include "/builtins/materials/skinning.glsl"
```

Comment:
I decided to use includes for the shared parts here, both to reduce the code but also to make sure we test incudes more in "production code" apart from the unit tests.